circus-train-integration-tests/src/test/data/com/hotels/bdp/circustrain/integration/CircusTrainHdfsHdfsIntegrationTest/partitioned-single-table-full-overwrite.yml

```diff
@@ -0,0 +1,9 @@
+table-replications:
```

Comment:
I think at some point we should think about removing all these duplicated yaml files and building them programmatically, just an idea

Reply:
The intention initially was for the integration tests to read in the yaml to ensure the yaml conversion code etc. is working. I don't think we envisioned there being this many integration tests, ideally we should only add them to test things that can't be tested in any other way.

Reply:
Yep understood, we could have a couple of template yaml files, and create temp files from these for each test on the fly

Reply:
Adrian suggested that I add an integration test class for the replication modes, so that will remove all these duplicated files 👍

Reply:
I think a separate task could be to evaluate the existing integration tests, clean them up so we have the minimal necessary and we could look into templated files as part of that. I'd suggest a separate PR though.